unit_test/download_test.py

```diff
@@ -5,7 +5,7 @@
 
 from cibuildwheel.util import download
 
-DOWNLOAD_URL = "https://raw.githubusercontent.com/joerick/cibuildwheel/v1.6.3/requirements-dev.txt"
+DOWNLOAD_URL = "https://raw.githubusercontent.com/pypa/cibuildwheel/v1.6.3/requirements-dev.txt"
```

Comment:
This one's not going to work until the move.

Reply:
Yeah... I think we'll merge this PR after the move has taken place.